cmd/npmExecuteEndToEndTests.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+func npmExecuteEndToEndTests(config npmExecuteEndToEndTestsOptions, _ *telemetry.CustomData) {
+	c := command.Command{}
+
+	c.Stdout(log.Writer())
+	c.Stderr(log.Writer())
+	runNpmExecuteEndToEndTests(config, &c)
+}
+
+func runNpmExecuteEndToEndTests(config npmExecuteEndToEndTestsOptions, c command.ExecRunner) {
+	env := os.Getenv("BRANCH_NAME")
```

Comment:
Use `orchestrator` package to retrieve current branch name

Reply:
[a21e7d2](https://github.com/SAP/jenkins-library/pull/5035/commits/a21e7d200ef39871ab4b9b30de03393fc6f49100)